website/docs/getting-started.md

```diff
@@ -4,191 +4,184 @@
 
 1. Deploy ORC to your Kubernetes cluster
 1. Create a Kubernetes secret containing a `clouds.yaml`
-1. Create an OpenStackCloud object pointing to that `clouds.yaml`
 1. Deploy your OpenStack infrastructure as Kubernetes custom resources
 
 ## Deploy ORC to your Kubernetes cluster
 
-From the git repository, run:
+From a fresh checkout of the git repository, from the `main` branch, run:
 
 ```sh
-make deploy IMG=quay.io/orc/openstack-resource-controller
+make deploy IMG=quay.io/orc/openstack-resource-controller:branch-main
 ```
 
 ## Create a Kubernetes secret containing a `clouds.yaml`
 
 ```sh
 kubectl create secret generic openstack-clouds \
-    --from-file=${XDG_CONFIG_HOME}/openstack/clouds.yaml
+    --from-file=${XDG_CONFIG_HOME:-~/.config}/openstack/clouds.yaml
```

Comment:
It's important that the key here be `clouds.yaml`. I'm pretty sure this command will work, but if the user decides to do something even slightly clever with their clouds.yaml, e.g. copy it to clouds.yaml.orc and remove bits they don't want to include, then it won't work because the key will be wrong.

Even though this is the quick start guide, I wonder if it's worth being explicit with the key mapping. I haven't looked up the CLI syntax to do that but I'm sure it's possible.